build/compiler.js

```diff
@@ -17,6 +17,7 @@ const ProgressBarPlugin = require('progress-bar-webpack-plugin');
 const webpackDevMiddleware = require('../lib/simple-webpack-dev-middleware');
 const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 const {
+  gfxWebpackPlugin,
```

Comment:
I think `zopfliWebpackPlugin` would be a more apt name since it indicates the purpose. It's hard to tell what `gfxWebpackPlugin` based on the name.